modules/op-running-as-rootless-in-container.adoc

```diff
@@ -0,0 +1,45 @@
+[id='op-running-as-rootless-in-container_{context}']
+= Running as rootless in container
+
+You can run as rootless inside a container. To run as a rootless user inside the container, you need your container to do some setup to add the `subuid`` and `subgid` with the latest shadow-utils.
+
```

Comment:
If this is a procedure topic, please add a line with `.Procedure` and rewrite the following information as numbered steps. Consider using separate procedure modules if you have multiple independent procedures.